Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,151 @@
+#Basic copy cat design of the mood file.
+#Figured I shouldn't fix something that isn't broken.
+#Shout out to whoever did it right the first time.
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imhaving",category=['you'],prompt="I'm Having",pool=True,unlocked=True))
+label monika_imhaving:
```

Comment:
This might make for a good introduction to the feature, but I think after a while, it might get a bit old for Monika to go through these exact lines every time. 

It seems like Monika is the one asking here, but I thought this topic was supposed to be more initiated by the player. 

It might seem a bit weird to tell her you're eating something out of nowhere, so I think there should be some input from Monika, but I think it could be done better. 

Maybe you could have this label be a random topic and an introduction to the feature and have another label for quickly telling her that you're eating something and have both call the database.